csp-mixin/signals/gcpceOverview.libsonnet

```diff
@@ -0,0 +1,284 @@
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      systemProblemCount: {
+        name: 'VM Availability',
+        description: 'Measure of availability of virtual machines',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          stackdriver: {
+            expr: 'count(sum by(instance_id)(stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count{%(queriesSelector)s}))',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      top5CpuUtilization: {
+        name: 'Top 5 Instances by CPU Utilitization',
+        description: 'Fractional utilization of allocated CPU on an instance',
+        type: 'gauge',
+        aggFunction: 'sum',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name', 'job', 'project_id'],
+            exprWrappers: [['topk(5, 100*', ')']],
+          },
+        },
+      },
+
+      top5SystemProblem: {
+        name: 'Top 5 Instances by System problem',
+        description: 'List of top 5 instances with system problems.',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'topk(5, sum by (instance_name, job, project_id) (increase(stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count{%(queriesSelector)s}[$__range])))',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+
+      top5DiskWrite: {
+        name: 'Top 5 Instances by Disk write bytes',
+        description: 'List of top 5 instances by disk write bytes',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'topk(5, sum by (instance_name, job, project_id) (increase(stackdriver_gce_instance_compute_googleapis_com_instance_disk_write_bytes_count{%(queriesSelector)s}[$__range]))\n)',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+
+      top5DiskRead: {
+        name: 'Top 5 Instances by Disk read bytes',
+        description: 'List of top 5 instances by disk read bytes',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'topk(5, sum by (instance_name, job, project_id) (increase(stackdriver_gce_instance_compute_googleapis_com_instance_disk_read_bytes_count{%(queriesSelector)s}[$__range]))\n)',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+
+      // Table
+      tableCpuUtilization: {
+        name: 'tableCpuUtilization',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: '100 * sum by (instance_name, job, project_id)(stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{job!="",job=~".+",project_id=~".+"})',
```

Comment:
don't you want to filter by job or project_id?

Reply:
Nice catch!